pandas/_config/config.py

```diff
@@ -460,9 +460,7 @@ def register_option(
     path = key.split(".")
 
     for k in path:
-        # NOTE: tokenize.Name is not a public constant
-        # error: Module has no attribute "Name"  [attr-defined]
-        if not re.match("^" + tokenize.Name + "$", k):  # type: ignore[attr-defined]
+        if not re.match("^" + tokenize.Name + "$", k):
```

Comment:
latest typeshed (bundled with mypy) exposes undocumented attributes

Reply:
does this mean theres a lower bound for what mypy we want?

Reply:
I think that in general it is recommended to pin mypy see https://mypy.readthedocs.io/en/stable/existing_code.html?highlight=pin#continuous-integration

> does this mean theres a lower bound for what mypy we want?

with warn_unused_ignores, yes.

Reply:
so I could change to mypy=0.782 (best practice) or mypy>=0.782 (see https://github.com/pandas-dev/pandas/issues/27568#issuecomment-528938502)

Reply:
commented above, prefer either of these to no version